common/middleware/add-user-information.js

```
@@ -0,0 +1,7 @@
+// adds relevant user information to res.locals for use by nunjucks templates
+
+module.exports = (req, res, next) => {
+  res.locals.username = req.session['X-Auth-Name'] || 'Test User'
```

If 'X-Auth-Name' is not present we should throw an error.